test/integration/scene-stream-access/add-scene-stream-access-handler.spec.ts

```diff
@@ -313,16 +299,244 @@ test('GET /scene-stream-access - gets streaming access for scenes', ({ component
+      streaming_key: mockSceneStreamAccess.streaming_key,
+      created_at: Number(mockSceneStreamAccess.created_at),
+      ends_at: Number(mockSceneStreamAccess.created_at) + FOUR_DAYS
     })
+  })
 
-    expect(response.status).toBe(400)
+  it('returns 200 with streaming access when user has world permission', async () => {
+    const { localFetch } = components
+
+    jest.spyOn(handlersUtils, 'validate').mockResolvedValueOnce(metadataWorld)
+    stubComponents.places.getPlace.resolves({
+      id: placeWorldId,
+      world_name: 'name.dcl.eth'
+    } as PlaceAttributes)
+    stubComponents.land.hasLandPermission.resolves(false)
+    stubComponents.world.hasWorldOwnerPermission.resolves(true)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'POST',
+        metadata: metadataWorld
+      },
+      owner
+    )
+
+    expect(response.status).toBe(200)
     const body = await response.json()
-    expect(body.message).toBe('Invalid Auth Chain')
+
+    if (body) {
+      const { ends_at, ...dbFields } = body
+      cleanup.trackInsert('scene_stream_access', dbFields)
+    }
+
+    expect(body).toEqual({
+      streaming_url: mockSceneStreamAccess.streaming_url,
+      streaming_key: mockSceneStreamAccess.streaming_key,
+      created_at: Number(mockSceneStreamAccess.created_at),
+      ends_at: Number(mockSceneStreamAccess.created_at) + FOUR_DAYS
+    })
+  })
+
+  it('returns 200 with streaming access when is an admin', async () => {
+    const { localFetch } = components
+
+    stubComponents.land.hasLandPermission.resolves(false)
+    stubComponents.world.hasWorldOwnerPermission.resolves(false)
+    stubComponents.sceneAdminManager.isAdmin.resolves(true)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'POST',
+        metadata: metadataLand
+      },
+      admin
+    )
+
+    expect(response.status).toBe(200)
+    const body = await response.json()
+
+    expect(body).toEqual({
+      streaming_url: mockSceneStreamAccess.streaming_url,
+      streaming_key: mockSceneStreamAccess.streaming_key,
+      created_at: Number(mockSceneStreamAccess.created_at),
+      ends_at: Number(mockSceneStreamAccess.created_at) + FOUR_DAYS
+    })
+  })
+
+  it('returns 401 when user is not authorized', async () => {
+    const { localFetch } = components
+
+    stubComponents.land.hasLandPermission.resolves(false)
+    stubComponents.world.hasWorldOwnerPermission.resolves(false)
+    stubComponents.sceneAdminManager.isAdmin.resolves(false)
+    stubComponents.sceneManager.hasPermissionPrivilege.resolves(false)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'POST',
+        metadata: metadataLand
+      },
+      nonOwner
+    )
+
+    expect(response.status).toBe(401)
+  })
+
+  it('returns 404 when streaming access creation fails', async () => {
```

Comment:
We should be returning a 500 when the streaming access creation fails, as this is something that should not happen. What do you think about changing it?